mex.go

```diff
@@ -72,6 +73,9 @@ func (mex *messageExchange) forwardPeerFrame(frame *Frame) error {
 		// Note: One slow reader processing a large request could stall the connection.
 		// If we see this, we need to increase the recvCh buffer size.
 		return GetContextError(mex.ctx.Err())
+	case connErr := <-mex.errCh:
+		// @aravindv: This is to unblock read because we saw a connection error
```

Comment:
Suggest changing this to:
```go
// Unblock readers/writers on connection error and forward the error.
```

we don't tend to have notes include the author's names in the codebase.